### PR TITLE
Own projects filter

### LINKED ISSUE #212: Add own projects to the filter

### PR DESCRIPTION
**Pull request by lpatel99**

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,122 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (arnauds5) [inline .idea/codeStyles/Project.xml]: .idea files shouldn't be added.

File: app/src/main/java/com/sdp13epfl2021/projmag/database/impl/firebase/FirebaseUserdataDatabase.kt [hunk 2]
```diff
@@ -258,8 +257,7 @@ class FirebaseUserdataDatabase @Inject constructor(
     ) {
         firestore
             .collection(USER_PROFILE)
-            .document(userID)
-            .get()
+            .document(userID).get()
             .addOnSuccessListener { document ->
```
Comment (arnauds5) [inline app/src/main/java/com/sdp13epfl2021/projmag/database/impl/firebase/FirebaseUserdataDatabase.kt:261]: Isn't it more readable to keep the .get() on another line ?

**Review by arnauds5 — APPROVED** [review]
except the .idea file, looks good to me

File: app/src/main/java/com/sdp13epfl2021/projmag/database/impl/firebase/FirebaseUserdataDatabase.kt (hunk 2 above)
Comment (lpatel99) [reply app/src/main/java/com/sdp13epfl2021/projmag/database/impl/firebase/FirebaseUserdataDatabase.kt:261]: yes it was to get rid of a code climate warning, I can put it back if you like

**Review by giogio21 — APPROVED** [review]
look good for me

**Pull request merged**